spec.bs

```diff
@@ -1902,6 +1935,65 @@ To <dfn>convert a modifier to a string</dfn> given a [=part/modifier=] |modifier
   1. Return the result of running [=canonicalize a hash=] given |strippedValue|.
 </div>
 
+<h2 id=integrating>Integrating with other specs</h2>
```

Comment:
In streams we called the counterpart section "Using streams in other specifications", which I kind of like. "Use in other specifications" or "Using URL patterns in other specifications" seem nice.

Reply:
Done. A parallel change for the subheadings wasn't obvious, but I at least changed the anchors.